libevm/EVMC.cpp

```diff
@@ -73,8 +73,14 @@ owning_bytes_ref EVMC::exec(u256& io_gas, ExtVMFace& _ext, const OnOpFunc& _onOp
         cwarn << "Execution rejected by EVMC, executing with default VM implementation";
         return VMFactory::create(VMKind::Legacy)->exec(io_gas, _ext, _onOp);
 
+    case EVMC_INTERNAL_ERROR:
```

Comment:
Why did you added this case?

Reply:
Only for clarity for the reader, but it is not necessary of course.

Reply:
Should we have a comment here that negative ones are internal errors while positive ones are non-internal?